lang/python/pluggy/patches/001-do-not-use-setuptools-scm.patch

```diff
@@ -0,0 +1,20 @@
+Index: pluggy-0.12.0/setup.py
+===================================================================
+--- pluggy-0.12.0.orig/setup.py
++++ pluggy-0.12.0/setup.py
```

Comment:
Would you please remove it the exact version?

Reply:
Removed

Reply:
Removed